src/vendor/cache/extensions/x86_64-linux/3.1.0/eventmachine-abe347b824e3/gem_make.out

```diff
@@ -0,0 +1,16 @@
+current directory: /home/ramonskie/workspace/bosh/bosh/src/vendor/cache/eventmachine-abe347b824e3/ext/fastfilereader
+/home/ramonskie/.rbenv/versions/3.1.0/bin/ruby -I /home/ramonskie/.rbenv/versions/3.1.0/lib/ruby/3.1.0 -r ./siteconf20220114-83401-11p8t.rb extconf.rb
+creating Makefile
+
+current directory: /home/ramonskie/workspace/bosh/bosh/src/vendor/cache/eventmachine-abe347b824e3/ext/fastfilereader
+make DESTDIR\= clean
+
+current directory: /home/ramonskie/workspace/bosh/bosh/src/vendor/cache/eventmachine-abe347b824e3/ext/fastfilereader
```

Comment:
We should probably ignore the src/vendor/cache/extensions directory. Will create a separate PR for that.